src/Pie.js

```diff
@@ -22,6 +22,7 @@ export default class Pie extends Viz {
     super();
 
     this._shapeConfig = assign(this._shapeConfig, {
+      ariaLabel: (d, i) =>  this._pieData ? `${++this._pieData[i].index}. ${this._drawLabel(d, i)}, ${d.value}.` : "",
```

Comment:
We cannot use `d.value` directly here, as the user may have defined their own accessor. This should be `this._value(d, i)`